packages/snap/src/stableToken.ts

```diff
@@ -0,0 +1,3 @@
+export const STABLE_TOKEN_ABI = JSON.parse(
```

Comment:
whats the benefit of this being a string that is parsed into JSON vs a json file vs straight up a ts object.

Reply:
+1 those options may be a bit more readable. You can check out the SocialConnect docs repo for an example of how to do this with json files https://github.com/celo-org/SocialConnect/blob/main/examples/constants.ts. I'm not sure about ts object but that could also work or potentially be cleaner

Reply:
I might also suggest putting all ABIs into a seperate directory